codegen/BUILD

```diff
@@ -8,6 +8,10 @@ py_binary(
         "//torch_xla/csrc:aten_xla_type.cpp",
         "@torch//:torchgen_deps",
     ],
+    deps = [
+        "@pypi_torch//:pkg",
```

Comment:
Why is this needed? Is this pulling pytorch whl from pypi?

Reply:
No, it is using whichever pytorch which is mentioned in requirements_lock.txt file. To support your workflow it will be locally built .whl file. For what its worth, it could pull pytorch from pypi as well, but I don't think it is what you need here.

Note this target has always been depending on pytorch, but non-hermetically (installed on the system) that is why the dependency was not explicitly defined. Now, since it is hermetic the dependency must be explicit (which is  good and idiomatic thing).